spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryTaskPlatformAutoConfiguration.java

```diff
@@ -38,6 +41,13 @@ public TaskPlatform cloudFoundryPlatform(CloudFoundryTaskPlatformFactory cloudFo
 		return cloudFoundryTaskPlatformFactory.createTaskPlatform();
 	}
 
+	@ConditionalOnCloudPlatform(CloudPlatform.CLOUD_FOUNDRY)
+	@Primary
```

Comment:
Use of `@Primary` like this is actually pointless as it exists on other beans as well. It would then fail if two primary beans exists.